server.go

```diff
@@ -604,6 +657,10 @@ func (s *server) Stop() error {
 
 	close(s.quit)
 
+	if s.torController != nil {
```

Comment:
Don't see where the controller was even started in the first place.

Reply:
See https://github.com/lightningnetwork/lnd/pull/1159/commits/0fb2bd98a130812fc0f2f5a9d0fc4acb0a2fe716#diff-34c6b408d72845d076d47126c29948d1R309.

Reply:
This has now been moved to `server.Start()`.